### PR TITLE
STYLE: Remove unnecessary " << " interruptions

### PR DESCRIPTION
**Pull request by dzenanz**
This was caused by style-related commit:
814a24c84d3bf4bdc42ce8f07276997fabdb45d3

This was accomplished by simple search and replace,
searching for `" << "` and replacing by an empty string.

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: Modules/ThirdParty/Eigen3/src/itkeigen/Eigen/src/Geometry/Quaternion.h [hunk 1]
```diff
@@ -216,7 +216,7 @@ class QuaternionBase : public RotationBase<Derived, 3>
 
 #ifndef EIGEN_NO_IO
   friend std::ostream& operator<<(std::ostream& s, const QuaternionBase<Derived>& q) {
-    s << q.x() << "i + " << q.y() << "j + " << q.z() << "k" << " + " << q.w();
+    s << q.x() << "i + " << q.y() << "j + " << q.z() << "k + " << q.w();
```
Comment (hjmjohnson) [inline Modules/ThirdParty/Eigen3/src/itkeigen/Eigen/src/Geometry/Quaternion.h:219]: Do not modify ThirdParty.

**Review by hjmjohnson — CHANGES_REQUESTED** [review]
Do not make changes to ThirdParty directories.

I assume that line length checks have been done.  It looks like perhaps the previous splitting may have been to keep line lengths short.  Be particularly careful of making lines too long in the Examples directory (used for documentation).

**Review by N-Dekker — APPROVED** [review]
- Reminds me of PR #3699 commit ab62a8eef421e89e61c3eed051105ec91e888eb1 😃 

Thanks @dzenanz Approved 👍 

Oh, I agree with Hans request for changes anyway 😸

**Comment by hjmjohnson** [comment]
@dzenanz Looks good other than ThirdParty changes.

**Review by N-Dekker — APPROVED** [review]

**Comment by dzenanz** [comment]
This code change removes characters, making lines shorter. Also, clang formatting is now invoked as part of pre-commit hook, right?

**Pull request merged**